src/main/java/com/yanna/stepanova/service/impl/BookServiceImpl.java

```diff
@@ -1,28 +1,68 @@
 package com.yanna.stepanova.service.impl;
 
+import com.yanna.stepanova.dto.BookDto;
+import com.yanna.stepanova.dto.CreateBookRequestDto;
+import com.yanna.stepanova.exception.EntityNotFoundException;
+import com.yanna.stepanova.mapper.BookMapper;
 import com.yanna.stepanova.model.Book;
+import com.yanna.stepanova.model.MyRandom;
 import com.yanna.stepanova.repository.BookRepository;
 import com.yanna.stepanova.service.BookService;
+import jakarta.transaction.Transactional;
 import java.util.List;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+@RequiredArgsConstructor
 @Service
 public class BookServiceImpl implements BookService {
     private final BookRepository bookRepo;
+    private final BookMapper bookMapper;
+    private final MyRandom myRandom;
```

Comment:
sorry, couldn't figure out: if write 'private final Random random' instead of MyRandom class - it can't initialize Random class and I have no idea how fix it

Reply:
please dont communicate here, all your questions should be asked at the Mate platforms chat